webpack/assets/javascripts/react_app/components/SearchBar/index.js

```diff
@@ -21,7 +21,7 @@ const SearchBar = ({
 }) => {
   const [search, setSearch] = useState(initialQuery || searchQuery || '');
   const { response, status, setAPIOptions } = useAPI('get', url, {
-    params: { search },
+    params: { ...apiParams, search },
```

Comment:
Should also be added a few lines down in `setAPIOptions` or change `setAPIOptions` to include the previous options